### PR TITLE
Fixes to WET to allow themes to install WET via npm

### PR DESCRIPTION
**Pull request by LaurentGoderre**

### CODE REVIEW

File: Gruntfile.coffee
Comment (nschonni) [inline Gruntfile.coffee]: Is this because we don't want to regenerate the sprites each time? What happens if the sprite map needs to change?

File: package.json
Comment (nschonni) [inline package.json]: `>=4.0.0` so people running 5+ on Node 8 don't get warnings

File: package.json
Comment (nschonni) [inline package.json]: Is this a regular dependency because of the recursion?

File: package.json
Comment (nschonni) [inline package.json]: What happens to the downstream projects that don't have bower as a dependency anymore

File: Gruntfile.coffee
Comment (LaurentGoderre) [reply Gruntfile.coffee]: This never targeted anything....that path doesn't exist

File: package.json
Comment (LaurentGoderre) [reply package.json]: I don't get warning and I am running npm 5

File: package.json
Comment (LaurentGoderre) [reply package.json]: Ooops, nop, copy/past mistake

File: package.json
Comment (LaurentGoderre) [reply package.json]: We need to get rid of bower altogether but this is still progress

**Review by nschonni — APPROVED** [review]
Sure, let's try it out

**Comment by LaurentGoderre** [comment]
@duboisp can we?

**Comment by duboisp** [comment]
@LaurentGoderre as I mentionned in #8279, PR can be merged outside of the Technical review meeting, it just needs to be justified about why it can’t wait. Fixing a current broken build, which was already been discussed at a technical review meeting, can be merged after it's review is approved.

**Pull request merged**
